buildSrc/src/main/java/Dependencies.kt

```diff
@@ -32,6 +32,8 @@ object Versions {
 object Dependencies {
     const val gradle_build_tool = "com.android.tools.build:gradle:${Versions.gradleBuildTool}"
     const val databinding = "androidx.databinding:compiler:${Versions.dataBinding}"
+    const val  appcompat= "androidx.appcompat:appcompat:1.0.0-beta01"
+    const val constraint ="androidx.constraintlayout:constraintlayout:1.1.3"
```

Comment:
```suggestion
    const val constraint = "androidx.constraintlayout:constraintlayout:1.1.3"
```
This version `1.1.3` should also be above in a variable like `appcompat`, like this:
```
const val appcompat = "1.1.3"
```

then you would use this like:
```
const val constraint ="androidx.constraintlayout:constraintlayout:${Versions.appcompat}"
```